.github/workflows/fs_write_full_disk.yml

```diff
@@ -0,0 +1,35 @@
+name: Fs write full disk
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Create disk image
+        run: |
+          fallocate -l 512K disk.img
+          mkfs disk.img
+
+      - name: Mount disk image
+        run: |
+          mkdir ./td
+          sudo mount -o loop disk.img ./td
+
+      - name: Set permissions
+        run: chmod a+wr ./td
+
+      # Add more steps for testing as needed
```

Comment:
Thanks for the PR, but this action is incomplete and hasn't undergone any real testing. Would you be willing to complete this part?

Reply:
Certainly! ，but I don't know specifically what is meant by "real testing."

Reply:
In this PR, you already setup a test env. We should write some rust code that do test to make sure our code works as expected.

For example:

```rust
#[tokio::test]
async fn test_file_write_on_full_disk() {
    let data = [96u8; 1024 * 512]; // The writable was smaller than 512KB
    op.write("/test", data.to_vec()).await.unwrap();
}
```